decoder/vaapidecoder_vc1.cpp

```diff
@@ -355,20 +383,32 @@ YamiStatus VaapiDecoderVC1::decode(uint8_t* data, uint32_t size, uint64_t pts)
         return YAMI_FAIL;
     }
 
-    if ((m_parser.m_frameHdr.picture_type == FRAME_I)
-        || (m_parser.m_frameHdr.picture_type == FRAME_P)
-        || (m_parser.m_frameHdr.picture_type == FRAME_SKIPPED)) {
-        m_forwardPicture = picture;
+    if (m_parser.m_frameHdr.picture_type == FRAME_B)
+        picture->m_isReference = false;
```

Comment:
the m_isRefernce is only use in this function, so why make it as member variable